### DISCUSSION TITLE
doris 从什么版本开始支持FQDN

### DISCUSSION THREAD

**Discussion by xuanshuangqi** [apache/doris#18461] [RFC]
1. doris dev的文档中已经有关于如何配置 FQDN 的信息了，但是我发现 support FQDN 的代码好像只有 在master分支上有，在1.2.3-rc02 以及 1.2的几个tag上都没有 ，代码应该是2022 年合并的，1.2.3-rc0 是2023年build的，是因为什么原因没有合并到这个tag中嘛？这其中有什么缘由嘛？

**Comment by LemonLiTree** [comment]
2.0

**Reply by xuanshuangqi** [reply]
多谢
